build/gulp/tasks/test-projects.tsx

```diff
@@ -16,11 +17,33 @@ const log = (context: string) => (message: string) => {
   console.log('='.repeat(80))
 }
 
-export const publishPackage = async () => {
-  const filename = tmp.tmpNameSync({ prefix: 'stardust-', postfix: '.tgz' })
-  await sh(`yarn pack --filename ${filename}`)
+const installStardustPackages = async (tmpDirectory: string) => {
+  const stardustPackages = lernaAliases({ sourceDirectory: false })
+  const packedPackages = await Promise.all(
+    Object.keys(stardustPackages).map(
+      async (packageName: string): Promise<{ packageName: string; filename: string }> => {
+        const filename = tmp.tmpNameSync({ prefix: `stardust-`, postfix: '.tgz' })
+        const directory = stardustPackages[packageName]
 
-  return filename
+        await runIn(directory)(`yarn pack --filename ${filename}`)
+
+        return { packageName, filename }
+      },
+    ),
+  )
+
+  const packageJsonName = path.resolve(tmpDirectory, 'package.json')
```

Comment:
name of the var is a bit misleading - could we use either `packageJsonFileName` or `packageJsonPath`?